maintenance/wikia/convertPhalanxStatsCharset.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * One-time script to convert charset of phalanx_stats to latin1
+ * The reason is all the connections in MW open with latin1 and cause improperly decoding
+ *
+ * @package MediaWiki
+ * @subpackage Maintanance
+ *
+ * @author Armon Rabiyan <armon at fandom.com>
+ *
+ */
+
+require_once( dirname( __FILE__ ) . '/../commandLine.inc' );
+
+if ( isset( $options['help'] ) ) {
+	die( "Usage: php convertPhalanxStatsCharset.php [--help] [--dry-run] [--verbose]
+	--dry-run       dry run
+	--verbose       be verbose
+	--help          you are reading it right now\n\n" );
+}
+
+$startTime = microtime( true );
+
+$dryRun = isset( $options['dry-run'] );
+
+$schema = 'specials';
+$table = 'phalanx_stats';
+$alterCharset = "ALTER TABLE $table CONVERT TO CHARACTER SET latin1;";
+
+if ( isset( $options['verbose'] ) ) {
+	echo( "Executing query on $schema.$table: $alterCharset ...\n" );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the parentheses from this "echo" call. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2041)